<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1620</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="pcDeviceFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="upArrowButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>250</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Up Arrow</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downArrowButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>325</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Down Arrow</string>
     </property>
    </widget>
    <widget class="QListWidget" name="display">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <property name="itemAlignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="sessionProgressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>390</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>470</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>470</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>470</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeElapsed">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>220</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>10</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>On/Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="contactLight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>21</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:blue</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="treatmentLight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>21</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:green</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="contactLostLight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>21</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:red</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="cTGDisplay" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="batteryCharge">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>490</y>
       <width>50</width>
       <height>95</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk{
	background-color: black;}</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="environmentManagerFrame">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="pcDeviceInfo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>400</width>
       <height>561</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <property name="itemAlignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pcDeviceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PC Device</string>
     </property>
    </widget>
    <widget class="QWidget" name="cTGPC" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>400</width>
       <height>560</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>300</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>20</width>
       <height>601</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="EEGLabel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>EEG Display</string>
     </property>
    </widget>
    <widget class="QLabel" name="ECLabel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>320</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Environment Controls</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode9">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode8">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode7">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode10">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode11">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>11</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode12">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode13">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>160</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>13</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode14">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>160</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>14</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode15">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>160</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>15</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode17">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>160</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>17</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode21">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>21</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode19">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>19</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode18">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>160</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>18</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode20">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eegNode16">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>160</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>16</string>
     </property>
    </widget>
    <widget class="QPushButton" name="batteryUseButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>350</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Battery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="batteryFullButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>380</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Batter Full</string>
     </property>
    </widget>
    <widget class="QPushButton" name="batteryEmptyButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>410</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Batter Empty</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
